### DISCUSSION TITLE
About `use.data` setter and getter extension

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#133] [General]
Is this correct `version.value++;`? Or better to use `version.value = version.value + 1;`?

**Comment by GregoryConrad** [comment]
I believe `.value++` will use the underlying `.value = .value + 1`, so it should be fine. (If you try it and it works, then you're all set).

**Reply by busslina** [reply]
Sorry, dumb question hehe

**Reply by GregoryConrad** [reply]
It’s a fine question!
